.env.jenkins

```diff
@@ -1,2 +1,2 @@
 TESTFAIRY_ENABLED=1
-WALLET_WIP_ENABLED=1
\ No newline at end of file
+WALLET_WIP_ENABLED=0
```

Comment:
Why is wallet disabled in this PR?

Reply:
Wallet is now always enebled. WIP features are not enabled.

Reply:
But i suppose we want to have WIP features in Jenkins' builds anyway. Or i misunderstand something?

Reply:
I think WIP features should be in Jenkins too